tensorflow/contrib/lite/examples/ios/simple/AppDelegate.m

```diff
@@ -14,13 +14,12 @@
 
 #import "AppDelegate.h"
 
-//#import "RunModelViewController.h"
+#import "RunModelViewController.h"
 
 @implementation AppDelegate
 
-- (BOOL)application:(UIApplication *)application
-    didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-  return YES;
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
```

Comment:
Apologies.. used to be you could just create up an object in a xib and it would work. You can get rid of all of these methods though and just leave it empty.